### ISSUE TITLE
drop down menu does't respond well

### ISSUE THREAD

**Issue by DeanLemans** [GrapheneOS/Camera#461]
Its very difficult to click the drop down button and sometimes its not even responding.

**Comment by MHShetty** [comment]
Yes, using the swipe down gesture would be recommended as it is far more accessible and easier to use compared to the pressing the drop-down at the far top of the screen. We could possibly increase the dropdown button's height or extend the touch target space beyond the visible dropdown UI in the upcoming releases to address this as an issue. Thanks a lot for reporting this issue @DeanLemans!
